runtime/src/lib.rs

```diff
@@ -290,48 +348,44 @@ parameter_types! {
     pub const ExistentialDeposit: u128 = 0;
     pub const TransferFee: u128 = 0;
     pub const CreationFee: u128 = 0;
-    pub const TransactionBaseFee: u128 = 1;
-    pub const TransactionByteFee: u128 = 0;
     pub const InitialMembersBalance: u32 = 2000;
 }
 
-impl balances::Trait for Runtime {
-    /// The type for recording an account's balance.
+impl pallet_balances::Trait for Runtime {
     type Balance = Balance;
-    /// What to do if an account's free balance gets zeroed.
-    type OnFreeBalanceZero = (Staking, Session);
-    /// What to do if a new account is created.
-    type OnNewAccount = (); // Indices; // disable use of Indices feature
-    /// The ubiquitous event type.
-    type Event = Event;
-
     type DustRemoval = ();
-    type TransferPayment = ();
+    type Event = Event;
     type ExistentialDeposit = ExistentialDeposit;
-    type TransferFee = TransferFee;
-    type CreationFee = CreationFee;
+    type AccountStore = System;
+}
+
+parameter_types! {
+    pub const TransactionByteFee: Balance = 1; // TODO: adjust fee
+    pub const TargetBlockFullness: Perquintill = Perquintill::from_percent(25);
+    pub AdjustmentVariable: Multiplier = Multiplier::saturating_from_rational(1, 100_000);
```

Comment:
I'm not sure if these values match for our expected total issuance or if it even matters.

Reply:
I've removed them in the latest PR. We will have to pick correct values when we choose correct fees and transactions.

Reply:
Yes as discussed in our call we will revisit all these runtime parameters :+1: